src/Client/components/submission/code-editor.tsx

```
@@ -1,222 +1,277 @@
+        try {
+            const response = await api.post("/Candidate/TestCode", payload);
+            if (response.status === 200) {
+                const data = await response.data;
+
+                const updated = formattedTestCases.map((testCase) => {
+                    const result = data.find(
+                        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                        (r: any) => r.testCaseId === testCase.testCaseId
+                    );
+                    let status = "pending";
+                    if (result && result.isAccepted) status = "success";
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)